src/confchange/testdata/joint_autoleave.json

```diff
@@ -0,0 +1,96 @@
+{

```

Comment:
Json is too verbose and not readable as plain text. 29 line of codes are translated into 96, 661 bytes to 2.11KB. 🤣